openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/CubeOpenShiftConfiguration.java

```diff
@@ -75,6 +76,7 @@
     private final String templateURL;
     private final String templateLabels;
     private final String templateParameters;
+    private final String templateParametersFile;
```

Comment:
CubeOpenshiftConfiguration already has template paramters. We don't need templatesParamtersFile too.
I'd say that OpenshiftResourceInstaller should depracted templateParmatersFile and just use templatesPrameters from configuration.